Packs/Base/Scripts/DeleteIndicatorRelationships/DeleteIndicatorRelationships.py

```diff
@@ -0,0 +1,27 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import traceback
+
+''' MAIN FUNCTION '''
+
+
+def main():
+    try:
+        ids = demisto.args().get('ids')
```

Comment:
did you test it manually with single and multiple ids?

Reply:
yes